serialization/physics.proto

```diff
@@ -79,6 +79,11 @@ message ContinuousTrajectory {
 }
 
 message DiscreteTrajectory {
+  // A marker to indicate that a fork doesn't have it's position tracked.
+  // Added in Groethendieck/Haas.
```

Comment:
Haar, I don’t think we will have a [Haas](https://en.wikipedia.org/wiki/Ruth_Haas) release.